build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1765,6 +1779,13 @@ message CacheCapabilities {
   // Note that this does not imply which if any compressors are supported by
   // the server at the gRPC level.
   repeated Compressor.Value supported_compressors = 6;
+
+  // Compressors supported for inlined data in
+  // [BatchUpdateBlobs][build.bazel.remote.execution.v2.ContentAddressableStorage.BatchUpdateBlobs]
+  // and
+  // [BatchReadBlobs][build.bazel.remote.execution.v2.ContentAddressableStorage.BatchReadBlobs]
```

Comment:
To me it's still not clear why we want to mention BatchReadBlobs here. In the case of batch reading, this feature is already self-negotiating.

Reply:
BatchReadBlobs has basic negotiation- the client will never get blobs in a compressed encoding that it didn't ask for. But by itself it doesn't allow the client to prioritise compressors.

My thought here was that if the server advertises the supported compressors ahead of time, then the client can pick its favourite from the advertised list and make a request for that. I suspect that most servers which implement this will support the same set of compressors for BatchReadBlobs and BatchUpdateBlobs, and we could therefore use the same field to cover both. Maybe it would be better to use separate fields though.

Alternatively, maybe we could just state that `BatchReadBlobsRequest.acceptable_compressors` are specified in the client's order of preference?

Reply:
I’m a fan of the latter.

Reply:
WFM- done. Renamed the capabilities field to `supported_batch_update_compressors`.